documentation/src/test/java/example/ParameterizedTestDemo.java

```diff
@@ -135,9 +136,9 @@ void testWithMultiArgMethodSource(String first, int second) {
 
 	// tag::CsvSource_example[]
 	@ParameterizedTest
-	@CsvSource({ "foo, 1", "bar, 2", "'baz, qux', 3" })
+	@CsvSource({ "foo, 1", "bar, 2", "'baz, qux', 3", "'', 4", ",5" })
 	void testWithCsvSource(String first, int second) {
-		assertNotNull(first);
+		assertNotNull(Objects.toString(first, "<null>"));
```

Comment:
> So perhaps it would be better and clearer to the user to move the empty and null use cases to separate test methods.

This.

I'll revert the tests and describe the special cases entirely in the text. Like done here: https://github.com/junit-team/junit5/pull/1016#issuecomment-322039840